packages/gestalt/src/Badge.test.js

```diff
@@ -8,9 +8,15 @@ it('Badge renders', () => {
   expect(tree).toMatchSnapshot();
 });
 
-it('should render with white text and blue background', () => {
-  const instance = create(<Badge text="Badge" />).root;
+it('should render with proper styling', () => {
```

Comment:
Can we add these tests for all supported colors? (You can DRY that out by defining an array of objects with the `type` and expected className, then iterate over that array calling the test on each item)

Reply:
This seems unnecessary if we have Visual regression tests, no?